tests/rule_based_profiler/test_rule_based_profiler.py

```diff
@@ -1211,3 +1272,48 @@ def test_list_profilers_in_cloud_mode(mock_profiler_store: mock.MagicMock):
 
     assert res == keys
     assert store.list_keys.called
+
+
+@mock.patch("great_expectations.data_context.data_context.DataContext")
+@mock.patch(
+    "great_expectations.rule_based_profiler.domain_builder.SimpleColumnSuffixDomainBuilder"
+)
+@mock.patch(
+    "great_expectations.rule_based_profiler.expectation_configuration_builder.DefaultExpectationConfigurationBuilder"
+)
+def test_add_rule(
+    mock_data_context: mock.MagicMock,
+    mock_domain_builder: mock.MagicMock,
+    mock_expectation_configuration_builder: mock.MagicMock,
+    sample_rule_dict: dict,
+):
+
+    profiler: RuleBasedProfiler = RuleBasedProfiler(
+        name="my_mocked_rbp", data_context=mock_data_context, config_version=1.0
```

Comment:
The RBP is not actually mocked right? Could we change the name just to prevent any confusion?